pkg/api/titond_api.go

```diff
@@ -1,33 +1,33 @@
 package api
 
 import (
-	"fmt"
-
 	"github.com/tokamak-network/tokamak-titond-backend/pkg/db"
 	"github.com/tokamak-network/tokamak-titond-backend/pkg/kubernetes"
-	"github.com/tokamak-network/tokamak-titond-backend/pkg/model"
 	"github.com/tokamak-network/tokamak-titond-backend/pkg/services"
+	"github.com/urfave/cli/v2"
 )
 
 type TitondAPI struct {
 	k8s         *kubernetes.Kubernetes
 	db          db.Client
 	fileManager services.IFIleManager
+	ctx         *cli.Context
 }
 
-func NewTitondAPI(k8s *kubernetes.Kubernetes, db db.Client, fileManager services.IFIleManager) *TitondAPI {
+func NewTitondAPI(k8s *kubernetes.Kubernetes, db db.Client, fileManager services.IFIleManager, ctx *cli.Context) *TitondAPI {
 	return &TitondAPI{
 		k8s,
 		db,
 		fileManager,
+		ctx,
 	}
 }
 
-func (t *TitondAPI) CreateNetwork(data *model.Network) *model.Network {
-	// t.fileManager.UploadContent("File_name_9", " New Content 9 ")
-	result, _ := t.db.CreateNetwork(data)
-	status, _ := t.k8s.GetPodStatus("default", "l2geth-0")
-	fmt.Println(status)
-
-	return result
+func (t *TitondAPI) Initialize() {
```

Comment:
Please make `Config` type of Titond and set args of `NewTitondAPI`.
Thank you!

Reply:
Thank you!
I updated!